pkg/controller/webhook/reconcile.go

```diff
@@ -58,7 +66,41 @@ func (w *Params) ReconcileResources(ctx context.Context, clientset kubernetes.In
 		if _, err := clientset.CoreV1().Secrets(w.Namespace).Update(ctx, webhookServerSecret, metav1.UpdateOptions{}); err != nil {
 			return err
 		}
+		UpdateOperatorPods(ctx, clientset)
 	}
 
 	return nil
 }
+
+func UpdateOperatorPods(ctx context.Context, clientset kubernetes.Interface) {
+	labels := metav1.ListOptions{
+		LabelSelector: "control-plane=elastic-operator",
+	}
+	//Get all the pods that are related to control-plane label.
+	pods, err := clientset.CoreV1().Pods("elastic-system").List(ctx, labels)
+	if err != nil {
+		return
+	}
+	for _, pod := range (*pods).Items {
+		UpdateOperatorPod(pod, ctx, clientset)
+	}
+}
+
+func UpdateOperatorPod(pod corev1.Pod, ctx context.Context, clientset kubernetes.Interface) {
+	if pod.Annotations == nil {
+		pod.Annotations = map[string]string{}
+	}
+	pod.Annotations[UpdateAnnotation] =
+		time.Now().Format(time.RFC3339Nano)
+	if _, err := clientset.CoreV1().Pods(pod.Namespace).Update(ctx, &pod, metav1.UpdateOptions{}); err != nil {
+		if errors.IsConflict(err) {
```

Comment:
I'm not sure it's really worth making a difference between conflicts and other kinds of errors. The reason is that this update will not be retried. I would just log this message at the error level, and remove the comment stating that conflicts are handled.